lib/cass_schema/version.rb

```diff
@@ -1,3 +1,3 @@
 module CassSchema
-  VERSION = "0.1.0"
+  VERSION = "0.1.1"
```

Comment:
Minor, but should there be a bigger bump since the api has changed a little (could the change in the name of the public setup method be considered a breaking change?)